src/solver/cmake/simulation.cmake

```diff
@@ -34,13 +34,14 @@ set(SRC_SIMULATION
 		simulation/common-eco-adq.cpp
 		simulation/common-hydro-remix.cpp
 		simulation/common-hydro-levels.cpp
-		simulation/common-dispatchable-margin.cpp
 		simulation/adequacy-draft.h
 		simulation/adequacy-draft.cpp
 		simulation/adequacy.h
 		simulation/adequacy.cpp
 		simulation/economy.h
 		simulation/economy.cpp
+		simulation/base_post_process.h

```

Comment:
Rename **base_post_process.h/.cpp** into **post_process.h/.cpp** (according to remark of @sylvlecl )